descriptors/ModuleDescriptor-template.json

```diff
@@ -18,6 +18,11 @@
           "pathPattern": "/marccat/search",
           "modulePermissions": ["configuration.all"],
           "permissionsRequired": []
+        },{
+          "methods": ["GET"],
+          "pathPattern": "/marccat/searchAuth",
+          "modulePermissions": ["configuration.all"],
+          "permissionsRequired": []
```

Comment:
Is there a reason why these endpoints don't require permissions?

Reply:
At the moment, we haven't any specific permissions for this module. We will add them in the future.

Reply:
> We will add them in the future

Before the honeysuckle release hopefully?

Reply:
Also, for `modulePermissions`, we should follow the principle of least privilege. As far as I can tell, mod-marccat only needs to execute `GET /configuration/entries`, but does not need to `POST` to this API or use the `/configurations/audit` API or the `/configurations/entries/{id}` API. For that reason, we should only grant the `configuration.entries.collection.get` permission in `modulePermissions` and not `configuration.all`. Even if we used most or all the permissions from an API in a dependent module, we should still grant them individually and not use an `all` permission since we should not automatically be granted permissions to new APIs added to a dependent module that we do not yet and may never integrate with.

Reply:
> honeysuckle

I don't know.

Reply:
> Also, for `modulePermissions`, we should follow the principle of least privilege. As far as I can tell, mod-marccat only needs to execute `GET /configuration/entries`, but does not need to `POST` to this API or use the `/configurations/audit` API or the `/configurations/entries/{id}` API. For that reason, we should only grant the `configuration.entries.collection.get` permission in `modulePermissions` and not `configuration.all`. Even if we used most or all the permissions from an API in a dependent module, we should still grant them individually and not use an `all` permission since we should not automatically be granted permissions to new APIs added to a dependent module that we do not yet and may never integrate with.


Thanks for the comment, we will change the permissions for all the APIS in the next PR.